### PR TITLE
e2e tests: a small step towards arm and docs tweaks

### PR DESCRIPTION
**Pull request by goodhoko**
A few last days I spent trying to run e2e tests on my M1 Mac. This PR brings 1) a small patch that brings us closer to that 2) Some minor docs tweaks.

This alone isn't enough to make the e2e tests run on arm machines, that will need far more work. I decided to not spend time on that now, deffered the effort to [this issue](https://github.com/trezor/trezor-suite/issues/3616) and pushed what I got so far with this PR.

RBC (review by commits) & DNS (do not squash) please .)

### CODE REVIEW

**Comment by goodhoko** [comment]
The [failing job](https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1166033077) doesn't seem to be related to changes in this PR. Might that be a temporary error fixable by a mere retry?

**Comment by tsusanka** [comment]
I've restarted and it seems to me that.

File: docker/suite/Dockerfile [hunk 1]
```diff
@@ -9,8 +9,9 @@ RUN apt-get update && apt-get -y --no-install-recommends install \
     ca-certificates \
     curl
 
-RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.4/gosu-$(dpkg --print-architecture)" \
-    && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.4/gosu-$(dpkg --print-architecture).asc" \
+# TODO: for M1 doesn't work
+RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-$(dpkg --print-architecture)" \
+    && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-$(dpkg --print-architecture).asc" \
     && chmod +x /usr/local/bin/gosu
```
Comment (tsusanka) [inline docker/suite/Dockerfile:15]: I am not even 100% sure we need that line. I am wondering what happens if you remove it altogether. The `docker/suite/Dockerfile` needs some love and will be addressed.

**Review by tsusanka — APPROVED** [review]
Mostly nit-picks, LGTM.

File: docker/suite/Dockerfile (hunk 1 above)
Comment (goodhoko) [reply docker/suite/Dockerfile:15]: Without it, running the image fails for unsufficient permissions. (I'd be sruprised if curl would be able to spit out executable file.)

> The docker/suite/Dockerfile needs some love and will be addressed.

Agree. For example, it's amost a perfect duplicate of `docker/test-runner/Dockerfile` modulo the base image. I considered that out of scope. (This PR already being quite far from the scope of the feature I'm [working on now](https://github.com/trezor/trezor-suite/pull/3590)).

Comment (goodhoko) [reply docker/suite/Dockerfile:15]: This PR is good from my side. If you're happy, @tsusanka, please resolve this thread and merge. (I'm not authorized to do that.)

**Pull request merged**